Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:14.04
```

Comment:
The only reason is habits, it's the image I use more often. I will test with 16.04 and add a new commit.